File: augast2020/Day2_Design_HashSet.java
import java.util.*;
class MyHashSet {

    /** Initialize your data structure here. */
    HashMap<Integer, Integer> map = new HashMap();
    public MyHashSet() {
        this.map = map;
    }
    
    public void add(int key) {
        if (this.map.get(key) == null)
        {
            this.map.put(key, key);
        }
        
    }
    
    public void remove(int key) {
        if(this.map.get(key) != null)
        {
            this.map.remove(key);
        }
    }
    
    /** Returns true if this set contains the specified element */
    public boolean contains(int key) {
        if(this.map.get(key) != null)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

/**
 * Your MyHashSet object will be instantiated and called as such:
 * MyHashSet obj = new MyHashSet();
 * obj.add(key);
 * obj.remove(key);
 * boolean param_3 = obj.contains(key);
 */

File: augast2020/Day2_Design_HashSet.py
class MyHashSet:
    def __init__(self):
        """
        Initialize your data structure here.
        """
        self.Map = {}
        

    def add(self, key: int) -> None:
        if not key in self.Map.keys():
            self.Map[key] = key
            
    def remove(self, key: int) -> None:
        if key in self.Map.keys():
            del self.Map[key]
            
    def contains(self, key: int) -> bool:
        """
        Returns true if this set contains the specified element
        """
        if key in self.Map.keys():
            return True
        


# Your MyHashSet object will be instantiated and called as such:
# obj = MyHashSet()
# obj.add(key)
# obj.remove(key)
# param_3 = obj.contains(key)